yarn.lock

```diff
@@ -692,6 +692,13 @@
   resolved "https://registry.yarnpkg.com/@discoveryjs/json-ext/-/json-ext-0.5.7.tgz#1d572bfbbe14b7704e0ba0f39b74815b84870d70"
   integrity sha512-dBVuXR082gk3jsFp7Rd/JI4kytwGHecnCoTtXFb7DB6CNHp4rg5k1bhg0nWdLGLnOV71lmDzGQaLMy8iPLY0pw==
 
+"@emnapi/runtime@^1.2.0":
```

Comment:
Why lock changes without dependency change ? Isn't there a --frozen-lockfile in the CI ?

Reply:
Nope, it's just an artefact (i added and removed a package)